src/pages/index.tsx

```diff
@@ -26,46 +25,62 @@ export default function HomePage() {
     }
     handleLoggedIn();
   }, [auth])
- 
-   let videoSection1= (
-<div className="overflow-x-scroll w-screen flex flex-row">
-          {creatorData1.map((video: {
-            id: string; imgUrl: string;
-          }, idx: {}) => {
-            return <Link href={`/video/${video.id}`} key={video.id + Math.random()}><Card id={idx} key={video.id} imgUrl={video.imgUrl} /></Link>;
-          })}
-        </div>
-   )
 
-   let videoSection2= (
-<div className="overflow-x-scroll w-screen flex flex-row">
-          {creatorData2.map((video: {
-            id: string; imgUrl: string;
-          }, idx: {}) => {
-            return <Link href={`/video/${video.id}`} key={video.id + Math.random()}><Card id={idx} key={video.id} imgUrl={video.imgUrl} /></Link>;
-          })}
-        </div>
-   )
+  let videoSection1 = (
+    <div className="overflow-x-scroll w-screen flex flex-row"
+      onClick={() => {
+        context.setCreator("Full Send Podcast");
+        context.setPayment("https://buy.stripe.com/test_9AQbLbdhD8Y5eYM3cf")
+        context.setImage("https://yt3.googleusercontent.com/5_TYQJ-59yU45NoK1GpQcRuov8OgZuwuSiS-0X8IveZI3QK_tKaiQxx9BHGqVkGogD08zY-txA=s900-c-k-c0x00ffffff-no-rj")
+      }} >
+      {creatorData1.map((video: {
+        id: string; imgUrl: string;
+      }, idx: {}) => {
+        return <Link href={`/video/${video.id}`} key={video.id + Math.random()}><Card id={idx} key={video.id} imgUrl={video.imgUrl} /></Link>;
+      })}
+    </div>
+  )
+
+  let videoSection2 = (
+    <div className="overflow-x-scroll w-screen flex flex-row"
+      onClick={() => {
+        context.setCreator("David Dobrik");
+        context.setPayment("https://buy.stripe.com/test_9AQbLb4L7del4k89AC")
+        context.setImage("https://yt3.googleusercontent.com/ytc/AL5GRJULlbEP4MO2qjUSF3X8op4Errek_FlWpoXfbZfRmA=s176-c-k-c0x00ffffff-no-rj")
+      }}
+    >
+      {creatorData2.map((video: {
+        id: string; imgUrl: string;
+      }, idx: {}) => {
+        return <Link href={`/video/${video.id}`} key={video.id + Math.random()}><Card id={idx} key={video.id} imgUrl={video.imgUrl} /></Link>;
+      })}
+    </div>
+  )
 
-   let videoSection3= (
-<div className="overflow-x-scroll w-screen flex flex-row">
-          {creatorData3.map((video: {
-            id: string; imgUrl: string;
-          }, idx: {}) => {
-            return <Link href={`/video/${video.id}`} key={video.id + Math.random()}><Card id={idx} key={video.id} imgUrl={video.imgUrl} /></Link>;
-          })}
-        </div>
-   )
+  let videoSection3 = (
+    <div className="overflow-x-scroll w-screen flex flex-row"
+      onClick={() => {
+        context.setCreator("PowerfulJRE");
+        context.setPayment("https://buy.stripe.com/test_28o2aB2CZ6PX9EsfZ8")
+        context.setImage("https://yt3.googleusercontent.com/ytc/AL5GRJXxw6Z2O6R2LSeCbDFhcBITwJiVxR7-tFOW663eXg=s176-c-k-c0x00ffffff-no-rj")
```

Comment:
Reputation 

Move all those hardcoded stuff to constants 

File lib/constant.ts

And make object with all those constats